packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -271,6 +271,47 @@ def test_vip_ipv6(dcos_api_session):
                     marathon.Network.USER, marathon.Network.USER, ipv6=True)
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    'container',
+    list(marathon.Container))
+def test_vip_port_mapping(dcos_api_session,
```

Comment:
isn't `port_mapping` get tested as part of `test_vip` BRIDGE mode?

Reply:
It's true, but it's another case. `test_vip` checks connectivity between two tasks, `origin` and `proxy`. Here we start the third task `pm` with port mapping on the host where `proxy` is running. For example,

1. `origin` task is running on 10.0.0.2 with `1.1.1.7:10001` VIP label
2. `proxy` task is running on 10.0.0.1
3. `pm` task is running on 10.0.0.1 on bridge network with port mapping, host port 10001

It means we have `1.1.1.7:10001` (vip) and `10.0.0.1:10001` (pm) on the same host and `1.1.1.7:10001` must always be our `origin` task.

Reply:
Sorry, I missed the fact that this test was added to support the fix. Thanks for explaining it.